assets/stylesheets/kitten/components/form/_location-input.scss

```diff
@@ -0,0 +1,160 @@
+///
+///   <div class="k-LocationInput">
+///     <div class="k-LocationInput__label">…</div>
+///     <div class="k-LocationInput__icon">
+///       <svg />
+///     </div>
+///     <input class="k-LocationInput__group
+///                   k-LocationInput__input
+///                   k-LocationInput__autocomplete" />
+///   </div>
+
+@mixin k-LocationInput {
+  $font: 'source-sans';
+  $label-font: 'source-sans-semi-bold';
+  $font-size: -1;
+  $line-height: 1.3;
+  $height: k-px-to-rem(50px);
+
+  $icon-margin: k-px-to-rem(17px);
+
+  // Color autocomplete
+  $color-mainText: map-get($k-colors, 'font-1');
+  $color-secondaryText: map-get($k-colors, 'font-2');
+  $border-autocomplete: map-get($k-colors, 'line-2');
+
+  // Base
+  $color: map-get($k-colors, 'font-1');
+  $border: map-get($k-colors, 'line-2');
+  $background: map-get($k-colors, 'background-1');
+
+  // Focus
+  $border-focus: map-get($k-colors, 'primary-1');
+
+  // Disabled
+  $color-disabled: #fff;
+  $border-disabled: #fff;
+  $background-disabled: map-get($k-colors, 'line-2');
+
+  // Placeholder
+  $color-placeholder: map-get($k-colors, 'font-2');
+
+  // Is-error
+  $color-is-error: map-get($k-colors, 'error');
+  $border-is-error: map-get($k-colors, 'error');
+
+  // Border radius Text - input
+  $border-text-input: k-px-to-rem(4px);
+
+  .k-LocationInput {
+    @include k-typographyFont($font);
+    @include k-typographyFontSize($font-size, $line-height);
+
+    .k-LocationInput__group {
+      position: relative;
+      width: 100%;
+    }
+
+    .k-LocationInput__label {
+      @include k-typographyFont($label-font);
+
+      color: $color;
+      margin-bottom: k-px-to-rem(20px);
+    }
+
+    .k-LocationInput__input {
+      position: relative;
+      display: block;
+      box-sizing: border-box;
+      outline: none;
+      padding: 0 k-px-to-rem(15px) 0 k-px-to-rem(37px);
+
+      width: 100%;
+      height: $height;
+      background: $background;
+      border: 1px solid $border;
+      border-radius: $border-text-input;
+      transition: color .2s, border-color .2s;
+
+      color: $color;
+
+      &::placeholder {
+        color: $color-placeholder;
+      }
+
+      &::-moz-placeholder {
+        color: $color-placeholder;
+      }
+
+      &:focus {
+        outline: none;
+        border-color: $border-focus;
+      }
+
+      &:disabled {
+        border-color: $border-disabled;
+        background-color: $background-disabled;
+        color: $color-disabled;
+
+        cursor: not-allowed;
+
+        &::placeholder {
+          opacity: 1;
+        }
+
+        &::-moz-placeholder {
+          opacity: 1;
+        }
+      }
+    }
+
+    &.is-error {
+      border-color: $border-is-error;
+      color: $color-is-error;
+
+      &:focus {
+        border-color: $border-focus;
+        color: $color;
+      }
+    }
+
+    .k-LocationInput__autocomplete {
+      border-color: $border-autocomplete !important;
+      border-top: 1px solid !important;
```

Comment:
OK, dans leur doc ils ont un "TODO allow users to override styles without important". Laissons comme ça.